<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MappingToolWindow</class>
 <widget class="QDialog" name="MappingToolWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MappingToolWindow</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="mainSizer">
     <property name="verticalSpacing">
      <number>4</number>
     </property>
     <item row="1" column="0">
      <widget class="QFrame" name="renderFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Map box</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3" rowstretch="0" columnstretch="0" rowminimumheight="0" columnminimumwidth="0">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Map list</string>
            </property>
            <property name="indent">
             <number>7</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="mapTableView">
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>50</number>
            </attribute>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>23</number>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>23</number>
            </attribute>
            <attribute name="verticalHeaderMinimumSectionSize">
             <number>15</number>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addFramesButton">
            <property name="text">
             <string>Add frames</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeFrameButton">
            <property name="text">
             <string>Remove Frame</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Textures list</string>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="indent">
             <number>7</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListView" name="texturesListWidget">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addTextureButton">
            <property name="text">
             <string>Add Texture</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeTextureButton">
            <property name="text">
             <string>Remove Texture</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QWidget" name="toolWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <widget class="QPushButton" name="showAll">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>23</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../Media/Icons/icons.qrc">
          <normaloff>:/16x16/show_all.png</normaloff>:/16x16/show_all.png</iconset>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QWidget" name="menusWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Media/Icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
